chains/v4/chains_dev.json

```diff
@@ -3913,5 +3913,135 @@
+                "precision": 12
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://rpc.mainnet.listen.io",
+                "name": "Listen Foundation 1 node"
+            },
+            {
+                "url": "wss://wss.mainnet.listen.io",
+                "name": "Listen Foundation 2 node"
+            }
+        ],
+        "types": {
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/listen.json",
+            "overridesCommon": true
+        },
+        "color": "linear-gradient(315deg, #DE8405 0%, #FFBB49 100%)",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Listen.svg",
+        "addressPrefix": 42
+    },
+    {
+        "chainId": "eacdd2d5b42de9769ccbb6e8d9013ab0d90ab105bf601d4aac53e874c145ec21",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Tanganika",
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "DHX",
+                "precision": 18,
+                "priceId": "datahighway"
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://tanganika.datahighway.com",
+                "name": "DataHighway node"
+            }
+        ],
+        "types": {
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/tanganika.json",
+            "overridesCommon": true
+        },
+        "color": "linear-gradient(315deg, #7D20BF 15.1%, #584DDB 57.81%, #51BEF9 94.27%)",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Tanganika.svg",
+        "addressPrefix": 33
+    },
+    {
+        "chainId": "35a06bfec2edf0ff4be89a6428ccd9ff5bd0167d618c5a0d4341f9600a458d14",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Bajun",
+        "assets": [
+            {
+                "assetId": 0,
+                "symbol": "BAJU",
+                "precision": 12
+            }
+        ],
+        "nodes": [
+            {
+                "url": "wss://rpc-parachain.bajun.network",
+                "name": "AjunaNetwork node"
+            }
+        ],
+        "types": {
+            "url": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/chains/v2/types/bajun.json",
+            "overridesCommon": true
+        },
+        "color": "linear-gradient(315deg, #DF5262 16.15%, #EC725D 53.13%, #F38861 75.32%, #3F3F3F 75.41%)",
+        "icon": "https://raw.githubusercontent.com/nova-wallet/nova-utils/master/icons/chains/white/Bajun.svg",
+        "addressPrefix": 1337
+    },
+    {
+        "chainId": "ca93a37c913a25fa8fdb33c7f738afc39379cb71d37874a16d4c091a5aef9f89",
+        "parentId": "b0a8d493285c2df73290dfb7e61f870f17b41801197a149ca93654499ea3dafe",
+        "name": "Imbue",
```

Comment:
As discussed on the meeting we should remove this network due to connection problems